amulet/sentry.py

```diff
@@ -192,16 +192,16 @@ def relation(self, from_rel, to_rel):
 
 # Possibly use to build out instead of having code in setup()?
 class Talisman(object):
-    def __init__(self, services, rel_sentry='relation-sentry', juju_env=None, timeout=300):
```

Comment:
I think we can remove the rel_sentry arg too, but, we should add **kw so as to not break existing external code that's passing those args. Unless we plan to make this 2.0?